tests/service/data/SyntaxTree/DotLambda/UnderscoreToFunctionNallWithSpaceAndUnitApplication.fs.bsl

```diff
@@ -17,18 +17,16 @@ ImplFile
                      None),
                   Named (SynIdent (myFunc, None), false, None, (1,4--1,10)),
                   None,
-                  DotLambda
-                    (App
-                       (NonAtomic, false, Ident MyMethodCall,
-                        Const (Unit, (1,28--1,30)), (1,15--1,30)), (1,13--1,30),
-                     { UnderscoreRange = (1,13--1,14)
-                       DotRange = (1,14--1,15) }), (1,4--1,10), Yes (1,0--1,30),
+                  App
+                    (NonAtomic, false,
+                     DotLambda
+                       (Ident MyMethodCall, (1,13--1,27),
+                        { UnderscoreRange = (1,13--1,14)
+                          DotRange = (1,14--1,15) }), Const (Unit, (1,28--1,30)),
+                     (1,13--1,30)), (1,4--1,10), Yes (1,0--1,30),
                   { LeadingKeyword = Let (1,0--1,3)
                     InlineKeyword = None
                     EqualsRange = Some (1,11--1,12) })], (1,0--1,30))],
           PreXmlDocEmpty, [], None, (1,0--1,30), { LeadingKeyword = None })],
       (true, true), { ConditionalDirectives = []
                       CodeComments = [] }, set []))
-
-(1,0)-(1,30) parse error Incomplete structured construct at or before this point in expression
-(1,13)-(1,14) parse error  _. shorthand syntax for lambda functions can only be used with atomic expressions. That means expressions with no whitespace unless enclosed in parentheses.
```

Comment:
This is a nonatomic `let myFunc = _.MyMethodCall ()`

It was previously failing with the error message in the diff, and now it passes.
I guess I could move the error into typechecking, if keeping things in the parser does not make sense.
(even though I think keeping it in the parser should be doable?)